stores/metadata.go

```diff
@@ -397,6 +410,89 @@ func (raw rawObject) toSlabSlice() (slice object.SlabSlice, _ error) {
 	return slice, nil
 }
 
+func (s *SQLStore) Bucket(_ context.Context, bucket string) (api.Bucket, error) {
+	var b dbBucket
+	err := s.db.
+		Model(&dbBucket{}).
+		Where("name = ?", bucket).
+		Take(&b).
+		Error
+	if errors.Is(err, gorm.ErrRecordNotFound) {
+		return api.Bucket{}, api.ErrBucketNotFound
+	} else if err != nil {
+		return api.Bucket{}, err
+	}
+	return api.Bucket{
+		CreationDate: b.CreatedAt.UTC(),
+		Name:         b.Name,
+	}, nil
+}
+
+func (s *SQLStore) CreateBucket(_ context.Context, bucket string) error {
```

Comment:
the `_` is correct here but doing it here makes the store inconsistent, should we update it everywhere?

Reply:
Changed it for consistency